.gitignore

```diff
@@ -60,6 +64,9 @@ gitpushall.sh
 main_fragment.html
 toc_fragment.html
 
+# man pages
+docs/*.1
```

Comment:
why? :) I like to see those in `git status`

Reply:
because they are auto-generated!? :)

Reply:
Not all of them. Only the `profanity-*.1`.
And also I add them for each "release commit" and remove them later. And I like to see that with `git status` :)

Reply:
> Not all of them. Only the `profanity-*.1`.

OK, fair point, I'll update the ignore to `profanity-*.1`, OK?

or do I have to remove it completely?

Reply:
Since i add them in each release commit I would prefer to remove this line completey. Just so I see this and don't miss anything.